aten/src/ATen/native/native_functions.yaml

```diff
@@ -3406,7 +3406,7 @@
 # that doesn't propagate gradients to non-present entries on sparse.
 - func: _sparse_addmm(Tensor self, Tensor sparse, Tensor dense, *, Scalar beta=1, Scalar alpha=1) -> Tensor
   use_c10_dispatcher: full
-  named_guard: False
+  supports_named_tensor: True
```

Comment:
Named tensors don't support sparse tensors, so I think it would be better if we just deleted the line.
```
In [5]: torch.empty(2, 3, layout=torch.sparse_coo, names=['x', 'y'])
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-5-0b4a27d2e557> in <module>
----> 1 torch.empty(2, 3, layout=torch.sparse_coo, names=['x', 'y'])

RuntimeError: NYI: named tensors only support strided layout
```

Reply:
Got it, deleted.